.github/workflows/builds.yml

```diff
@@ -2,7 +2,7 @@ name: Builds
 on:
   push:
     branches:
-      - latest-edge
+      - v2-edge
```

Comment:
I see in the new branch `v2-edge` this already is at `v2-edge` but for the `latest-edge` branch (this one) shouldn't this stay at `latest-edge`?
Otherwise I assume the workflow will never run for this branch or do we intent to rename it to `v2-edge`?

Reply:
good spot, wrong branch, trying to two things at once badly :)